### PR TITLE
ACP2E-1122: [Documentation] "Allow Multiple Boxes for Shipping" is visible only "Qty Uses Decimals" is set to Yes.

### PR DESCRIPTION
**Pull request by vkolesny**
## Purpose of this pull request

_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

As per https://jira.corp.adobe.com/browse/ACP2E-992 we need to update the documentation.

"Allow Multiple Boxes for Shipping" is visible only when "Qty Uses Decimals" is set to Yes.

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

- Magento 2.4.x

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- No.

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- No.

## Affected documentation pages

_List HTML links for affected pages on <https://docs.magento.com/user-guide/>._

- https://experienceleague.adobe.com/docs/commerce-admin/inventory/configuration/configuration.html
- https://experienceleague.adobe.com/docs/commerce-admin/inventory/configuration/product-options.html

### CODE REVIEW

File: help/inventory-management/configuration.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Another way you can use the stock availability threshold is to manage products t
 | [!UICONTROL Minimum Qty Allowed in Shopping Cart] | Sets the minimum quantity of the product that can be purchased in a single order. |
 | [!UICONTROL Maximum Qty Allowed in Shopping Cart] | Sets the maximum quantity of the product that can be purchased in a single order. |
 | [!UICONTROL Qty Uses Decimals] | Allows decimal amounts, instead of whole numbers, for the quantity of a product. This setting is helpful for products sold by weight, volume, or length. Specified on the level of Source, calculated on the Stock level based on assigned Sources. |
-| [!UICONTROL Allow Multiple Boxes for Shipping] | Determines if parts of a product can be shipped separately. |
+| [!UICONTROL Allow Multiple Boxes for Shipping] | Determines if parts of a product can be shipped separately. Is visible when **Qty Uses Decimals** = `Yes`. |
```
Comment (leslietilling) [inline help/inventory-management/configuration.md:34]:
```suggestion
| [!UICONTROL Allow Multiple Boxes for Shipping] | Determines if parts of a product can be shipped separately. This option is visible when **[!UICONTROL Qty Uses Decimals]** = `Yes`. |
```

File: help/inventory-management/product-options.md [hunk 2]
```diff
@@ -77,7 +77,7 @@ Configure the quantities and additional settings per [added source](sources-add.
 
 1. Set **[!UICONTROL Qty uses Decimals]** to `Yes` if customers can use a decimal value rather than a whole number when entering the quantity ordered.
 
-1. Set **[!UICONTROL Allow Multiple Boxes for Shipping]** to `Yes` if the product can be sold separately, in many boxes.
+1. Set **[!UICONTROL Allow Multiple Boxes for Shipping]** to `Yes` if the product can be sold separately, in many boxes. This option is visible when **Qty Uses Decimals** is set to `Yes` only.
```
Comment (leslietilling) [inline help/inventory-management/product-options.md:80]:
```suggestion
1. Set **[!UICONTROL Allow Multiple Boxes for Shipping]** to `Yes` if the product can be sold separately, in many boxes. This option is visible when **[!UICONTROL Qty Uses Decimals]** is set to `Yes` only.
```

File: help/inventory-management/product-options.md [hunk 3]
```diff
@@ -104,7 +104,7 @@ Configure the quantities and additional settings per [added source](sources-add.
 | [!UICONTROL Minimum Qty Allowed in Shopping Cart] | Global | Determines the minimum number of the product that can be purchased in a single order. |
 | [!UICONTROL Maximum Qty Allowed in Shopping Cart] | Global | Determines the maximum number of the product that can be purchased in a single order. |
 | [!UICONTROL Qty Uses Decimals] | Global | Determines if customers can use a decimal value rather than a whole number when entering the quantity ordered. Options:<br />**[!UICONTROL Yes]** - Permits values to be entered as decimals, rather than whole numbers. Decimals are suitable for products sold by weight, volume, or length.<br />**[!UICONTROL No]** - Requires quantity values to be entered as whole numbers. |
-| [!UICONTROL Allow Multiple Boxes for Shipping] | Global | Determines if parts of the product can be shipped separately. |
+| [!UICONTROL Allow Multiple Boxes for Shipping] | Global | Determines if parts of the product can be shipped separately. Is visible when **Qty Uses Decimals** = `Yes`. |
```
Comment (leslietilling) [inline help/inventory-management/product-options.md:107]:
```suggestion
| [!UICONTROL Allow Multiple Boxes for Shipping] | Global | Determines if parts of the product can be shipped separately. This option is visible when **[!UICONTROL Qty Uses Decimals]** = `Yes`. |
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Minor edits and add ExL tags to UI labels for localization support.

File: help/inventory-management/configuration.md (hunk 1 above)
Comment (vkolesny) [reply help/inventory-management/configuration.md]: done

File: help/inventory-management/product-options.md (hunk 2 above)
Comment (vkolesny) [reply help/inventory-management/product-options.md]: done

File: help/inventory-management/product-options.md (hunk 3 above)
Comment (vkolesny) [reply help/inventory-management/product-options.md]: done

**Comment by vkolesny** [comment]
Minor edits were successfully added.

**Review by leslietilling — APPROVED** [review]

**Pull request merged**